### ISSUE TITLE
How do I find entropy bits?

### ISSUE THREAD

**Issue by nitrocode** [dwolfhub/zxcvbn-python#19]
I noticed that on Dropbox's introduction page of [zxcvbn](https://blogs.dropbox.com/tech/2012/04/zxcvbn-realistic-password-strength-estimation/) and their original [python-zxcvbn](https://github.com/dropbox/python-zxcvbn) contain a key `entropy`.

Is there any interest in having some of the entropy functions for [scoring.py](https://github.com/dropbox/python-zxcvbn/blob/bc1c2d32d92dc83a384da446edd7ca54e71eb9d9/zxcvbn/scoring.py#L43) ported over to this project?

**Comment by dwolfhub** [comment]
Hello!

The library has come a long way, and the author's of the zxcvbn project no longer do the calculation based on entropy. It's instead determined based on the number of guesses required to come up with the password using a brute force attack.

I'm probably misrepresenting their ideas, though. I did not write the logic for this library. I simply ported it over from JS to Python and aim to keep it 100% compatible with that version.

Check out [that repo](https://github.com/dwolfhub/zxcvbn-python/issues/19) for more information.

Cheers!
